assets/js/components/sell-quick-start.component.js

```diff
@@ -144,6 +150,8 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
       if (r.absoluteFeeBounds[0] === 0) {
         $scope.error['moreThanInWallet'] = true;
         $scope.offerUseAll();
+      } else {
+        $scope.status = {};
```

Comment:
Each quote triggers the building of a payment and checking for a fee of zero. I'm setting `status.busy = true` at the beginning of the function, and if the fee is _not_ zero, it clears the status and activates the button to enter sell flow. If fee _is_ zero, it offers the user the ability to use all minus the fee while keeping `status.busy`. There's gotta be a better way to do this without building a payment each time, since that seems to be an expensive operation that slows things down.